crates/sui-storage/src/node_sync_store.rs

```diff
@@ -15,25 +18,55 @@ use typed_store_macros::DBMapUtils;
 
 use tracing::trace;
 
+#[cfg(test)]
+use std::sync::Arc;
+
 /// NodeSyncStore store is used by nodes to store downloaded objects (certs, etc) that have
 /// not yet been applied to the node's SuiDataStore.
 #[derive(DBMapUtils)]
 pub struct NodeSyncStore {
-    /// Certificates/Effects that have been fetched from remote validators, but not sequenced.
     certs_and_fx: DBMap<TransactionDigest, (CertifiedTransaction, SignedTransactionEffects)>,
 
+    /// Certificates that have been fetched from remote validators, but not sequenced.
+    certs: DBMap<TransactionDigest, CertifiedTransaction>,
```

Comment:
Why do we need both certs and certs_and_fx? Also, is there a reason we cannot just use the certs table in authority store?

Reply:
Sorry, we don't - meant to delete the previous table.

We could use the certs table in the authority store for this - I was a little worried about the possibility of confusion when there is some other random process mutating the authority db, but it wouldn't be a correctness issue.

Reply:
Cleaned all of this up.